src/Error/CSException.php

```diff
@@ -28,6 +28,7 @@
  * @license   https://github.com/contentstack/contentstack-php/blob/master/LICENSE.txt MIT Licence
  * @link      https://pear.php.net/package/contentstack
  * */
+#[\AllowDynamicProperties]
 class CSException extends \Exception
```

Comment:
\ before exception is part of code ?

Reply:
The \ (backslash) character in front of AllowDynamicProperties indicates that it is a namespaced attribute. In PHP, namespaces are used to organize and group classes, functions, and constants to avoid naming conflicts.